src/app/components/graph/graph-viz.js

```diff
@@ -206,6 +206,20 @@ angular
                     },
                     show: true,
                 },
+                {
+                    id: 'export-png',
+                    content: 'Export PNG',
+                    selector: 'node',
+                    onClickFunction: function(event) {
+                        var options = {
+                            bg: '#005e7a'
+                        };
+                        var png64 = cy.png(options);
+                        var url = png64.replace(/^data:image\/[^;]+/, 'data:application/octet-stream');
+                        window.open(url);
```

Comment:
This is a cool approach! Seems to work well, but it looks like the file is downloaded with the named `download` on Chrome? What do you think about using a link instead? I think something like this should work:

```
var link = document.createElement('a');
link.download = 'dbt-dag.png';  // sets the filename for the download
link.href = url;
link.click();
```

Reply:
Yes, I wasn't sure about the browser support (~82% support: https://caniuse.com/#feat=download), but I guess it's progressive enhancement. If the browser doesn't support it, it will use the default filename. There is some crash in Edge13 apparently, but that's probably not a big percentage.